packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -558,6 +558,23 @@
                     }
                 }
             }
+            &-ctrader {
+                &-container {
+                    display: flex;
+                    flex-direction: column;
+                    margin-top: 4rem;
+                    margin-bottom: 6rem;
+                    @include mobile {
+                        margin-top: 5rem;
+                        margin-bottom: 0;
```

Comment:
```suggestion
margin: 5rem 0 0 0;
```

Reply:
We can do it later because we need to remove it after apps are there